gae_discourse_client/content.py

```diff
@@ -0,0 +1,48 @@
+"""Module for accessing and creating content through the Discourse API"""
+
+from google.appengine.ext import ndb
+
+
+class Error(Exception):
+    pass
+
+
+class ContentClient(object):
+    """An API client for interacting with Discourse for content-related actions"""
+
+    def __init__(self, api_client, category_client):
+        self._api_client = api_client
+        self._category_client = category_client
+
+    @ndb.tasklet
+    def get(self, category_id=None, page=0):
```

Comment:
Since this class doesn't directly map to a resource as written, `get` as a name here is ambiguous. I would go one of two ways:
   * Make the name more precise, e.g. `getTopics`. (You should probably do this anyway if you're not retrieving the category directly, but rather objects inside the category.)
   * Have separate clients for category and topic.